ArduCopter/AP_Arming.cpp

```diff
@@ -195,6 +195,14 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
             check_failed(ARMING_CHECK_PARAMETERS, display_failure, "Heli motors checks failed");
             return false;
         }
+
+        char fail_msg[50];
+        // check input mangager parameters
+        if (!copter.input_manager.parameter_check(fail_msg, ARRAY_SIZE(fail_msg))) {
+            check_failed(ARMING_CHECK_PARAMETERS, display_failure, " %s", fail_msg);
```

Comment:
Extra space in the format string.
```suggestion
            check_failed(ARMING_CHECK_PARAMETERS, display_failure, "%s", fail_msg);
```

Reply:
that was intentional to put a space between the "Pre-arm:" and the message.  is that wrong?

Reply:
got it.  Fixed!